packages/components/src/globals/grid/__tests__/__snapshots__/grid-test.js.snap

```diff
@@ -17,7 +16,7 @@ h2 {
 h3 {
   font-size: 1.75rem;
   font-weight: 400;
-  line-height: 1.28572;
+  line-height: 1.29;
```

Comment:
I think you may need to rebuild and then update snapshots, some of these changes seem to be unrelated